### PR TITLE
Add module-variables() and module-functions()

### PR DESCRIPTION
**Pull request by nex3**
Partially addresses #554

See sass/sass-spec#1403

### CODE REVIEW

**Comment by nshahan** [comment]
Does this travis run need attention?

**Comment by nex3** [comment]
It's failing because of the issue you pointed out in the specs. I've restarted it, so it should work now.

File: lib/src/visitor/async_evaluate.dart [hunk 1]
```diff
@@ -282,6 +282,32 @@ class _EvaluateVisitor
...
+      BuiltInCallable("module-variables", r"$module", (arguments) {
+        var namespace = arguments[0].assertString("module");
+        var module = _environment.modules[namespace.text];
+        if (module == null) {
+          throw 'There is no module with namespace "${namespace.text}".';
+        }
+
+        return SassMap({
+          for (var entry in module.variables.entries)
+            SassString(entry.key): entry.value
```
Comment (nshahan) [inline lib/src/visitor/async_evaluate.dart]: Any danger of exposing something you don't want to by returning this `Value` through the map without reboxing it in some way?

Comment (nex3) [reply lib/src/visitor/async_evaluate.dart]: No, `Value`s are intentionally only ever normal Sass values.

**Pull request merged**